datafusion/physical-expr/src/string_expressions.rs

```diff
@@ -50,43 +53,6 @@ macro_rules! downcast_string_arg {
     }};
 }
 
-macro_rules! downcast_primitive_array_arg {
```

Comment:
I reviewed the changes in this module carefully and they look good to me -- thank you @retikulum